subsys/bluetooth/host/gatt.c

```diff
@@ -3147,8 +3154,12 @@ static int bt_gatt_clear_ccc(u8_t id, const bt_addr_le_t *addr)
 				       (bt_addr_le_t *)addr, NULL);
 	}
 
+	struct addr_with_id addr_with_id = {
+		.addr = addr,
+		.id = id,
+	};
```

Comment:
iirc it is against the codying style to declare a variable in the middle of a function/statement scope, at least in Bluetooth we don't do that.

Reply:
I checked it before and there is no coding standard that would forbid it. I will however change it if you like to match the style of other files in bluetooth.

Reply:
Id prefer we keep the declaration in the beginning, at least I don't see anything against doing it and we keep the code a little bit more 'organized'.

Reply:
Fixed